ks8500_runner/changelog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [unreleased]
```

Comment:
You need to define a version number to be tracked on the global version.

Reply:
Not sure exactly what is allowed for versioning of a component. I added a 4 digit number to allow tracking the wrapped runner (first 3 digits) and changes to the Open Nebula Component (last digit). Let me know if this is not acceptable